### DISCUSSION TITLE
Quarkus cache: Use both Caffeine and Redis cache

### DISCUSSION THREAD

**Discussion by SivaM07** [quarkusio/quarkus#43802] [Q&A]
I have an usecase where certain cache has to maintained in-memory (caffeine) and some on Redis from the same application.

As far as I have debugged it looks like only one can be provided as quarkus.cache.type at buildtime

Is it the expected functionality ? Is there a possible way to achieve my usecase here ?

[a comment by an automated account is not shown]

**Reply by cescoffier** [reply]
Yes, unfortunately, at the moment, we support a single cache implementation. Please open a request for enhancements.

**Reply by SivaM07** [reply]
Is there a place where we can look at planned enhancements for cache and request for something if it is not planned already.

**Reply by emmanuelbernard** [reply]
@cescoffier it's quite a common problem we hear and maybe that's warranting an ADR on how you solve the "want to use two instances of X". So that resources like that know how to design and expose the flexibility while still keeping the simple use case less verbose.

**Reply by emmanuelbernard** [reply]
Probably the DB level model is the one? not sure.
